### ISSUE TITLE
Assistance with setting up the Envoy Logger as a service in Ubuntu Server 24.04 LTS

### ISSUE THREAD

**Issue by uselessinf0** [amykyta3/enphase-power-logger#12]
Hi Alex,

This is a great project. I don't have my Enphase system installed yet but I'm pretty excited and getting stuff ready and looking forward to utilizing your Envoy Logger project. So this is what I completed so far.

1. Installed Ubuntu Server 24.04 LTS in a Proxmox VM
2. Installed the latest Ubuntu updates
3. Installed the complete Python 3 environment - sudo python3-full
4.  Installed the required modules

sudo apt install python3-pip
sudo apt install python3-requests
sudo apt install python3-appdirs
sudo apt install python3-influxdb-client
sudo apt install python3-yaml
sudo apt install python3-certifi

5.  Created a service account "envoylog" in the Ubuntu instance (I don't have an Enphase system so it seems I can't create the service account user in the Enlighten App just yet)
6. Set 700 permissions to /home/eventlog
7.  Installed and Configure envoy_logger Under the Service Account

pip3 install --user git+https://github.com/amykyta3/envoy-logger --break-system-packages

To do once I have my Enphase system is installed:
=================================
1. Create service account in the Elighten app
2.  Finalize configuration of /home/envoylog/config.yml
3. Locally test that the logging script can read from Envoy, and push data to InfluxDB - I haven't installed InfluxDB yet, I guessing I should install it now? Should I have installed it  before step 7 listed above?
4.  Log into InfluxDB and verify data
5. Create a new Systemd service in the Ubuntu instance
6. Start the service and enable the service to start at boot time
7. Install Grafana on Ubuntu?? - does it make sense to to install Grafana on Ubuntu?
9. Add a connection to InfluxDB
10. Start building dashboards - use your query examples?

Do the Items I've completed so far look good? How does the "To do" list look?

Kind Regards,
Paul

**Comment by amykyta3** [comment]
Yep that seems to cover most steps.
Regarding where to install Grafana is up to you and what your plans are for hosting and presenting the data. A local install is a great way to start and experiment. 

Some more details here: https://github.com/amykyta3/envoy-logger/blob/main/docs/Setup-Instructions.md

**Comment by uselessinf0** [comment]
Thank you Alex.

Should I have installed InfluxDB before installing the Envoy Logger from the repository or does it not matter?

**Comment by amykyta3** [comment]
Yes. The logger requires an active instance of InfluxDB to be running.
